### PR TITLE
More random cleanups

### PR DESCRIPTION
**Pull request by FlorianLudwig**

### CODE REVIEW

File: hbmqtt/client.py [hunk 1]
```diff
@@ -498,7 +497,7 @@ def _initsession(
             broker_conf['cleansession'] = cleansession
 
         for key in ['uri']:
```
Comment (HerrMuellerluedenscheid) [inline hbmqtt/client.py:499]: Wouldn't it be cleaner to drop this iteration over one item and replace it with:
```
            if not broker_conf.get('uri'):
                raise ClientException("Missing connection parameter 'uri'")
```

Comment (FlorianLudwig) [reply hbmqtt/client.py:499]: this code indeed looks odd.

Will clean up that as well

File: hbmqtt/broker.py [hunk 2]
```diff
@@ -1,11 +1,13 @@
 # Copyright (c) 2015 Nicolas JOUANIN
 #
 # See the file license.txt for copying permission.
+from __future__ import annotations
```
Comment (FlorianLudwig) [inline hbmqtt/broker.py:4]: so, this was not in python 3.6, bummer.

File: hbmqtt/client.py (hunk 1 above)
Comment (FlorianLudwig) [reply hbmqtt/client.py:499]: `broker_conf['uri']` is not even use for the connection :sweat:

Comment (HerrMuellerluedenscheid) [reply hbmqtt/client.py:499]: Strange. So dump it I'd say.

Comment (HerrMuellerluedenscheid) [reply hbmqtt/client.py:499]: Or just leave it in place to avoid bad surprises.

**Comment by HerrMuellerluedenscheid** [comment]
@FlorianLudwig I'm afraid this needs a rebase :/ Sorry again for the `black` mess. When this is in I'd like to move forward with #34

**Comment by HerrMuellerluedenscheid** [comment]
LGTM

**Review by HerrMuellerluedenscheid — APPROVED** [review]

**Pull request merged**